### PR TITLE
Issue #7 lower the weight of the module

### PR DESCRIPTION
**Pull request by legovaer**

### CODE REVIEW

File: drupal_symfony_inject.install [hunk 1]
```diff
@@ -5,6 +5,16 @@
...
 
 define('SYMFONY_CLASS_LOADER_PROXIES_DIR', 'symfony_class_loader_proxies_dir');
+
+/**
+ * Implements hook_enable().
+ */
+function drupal_symfony_inject_enable() {
+  // Set a low module weight to ensure this module is enabled before those that
+  // depend on it.
+  db_query("UPDATE {system} SET weight = -99 WHERE name = 'drupal_symfony_inject' AND type = 'module' AND weight = 0");
```
Comment (sharique) [inline drupal_symfony_inject.install:15]: I think we don't need weight in where condition.

Comment (legovaer) [inline drupal_symfony_inject.install:15]: We do if the user has already enabled / disabled this module in the past. It won't be necessary to re-update the weight a second time.

Comment (vijaycs85) [inline drupal_symfony_inject.install:15]: It's good to go.

Comment (Ishino) [inline drupal_symfony_inject.install:15]: Looks good to me.

**Pull request merged**